app/services/slack_notification_service.rb

```diff
@@ -40,6 +79,19 @@ def filter_on_hold_labeled_action
     slack_bot.delete_message matches
   end
 
+  def channel(params)
+    lang = LANGUAGES[params.dig("repository", "language")&.to_sym]
+    repository_info = params.dig("repository")
+
+    if lang
+      return js_channels(repository_info, lang) if lang == 'javascript' || lang == 'typescript'
+
+      "##{lang}-reviewers"
```

Comment:
I would go with something like `{lang}-code-review` to avoid sending all the PRs to the same channel we already use to ask for codeowners or similar in our projects